### ISSUE TITLE
Issue to contact API

### ISSUE THREAD

**Issue by LCHARRE** [hzi-braunschweig/SORMAS-Project#2988]
We can't contact API. We test to login on api with curl:

# curl -u userTest:XXXXX http://172.18.0.5:6048/sormas-rest/communities/uuids

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html>
<head>
    <title>Login</title>
<script type="text/javascript">

    if (document.getElementById('layout-doc') != null) {
        // Just refresh the page... login will take over
        window.location = window.location;
    }
</script>
    <style type="text/css">
        /* clickjacking defense */
        body { display : none; }
    </style>
<link rel="stylesheet" type="text/css" href="/theme/com/sun/webui/jsf/suntheme/css/css_master.css" />
<script type="text/javascript">
djConfig={
    "isDebug": false,
    "debugAtAllCosts": false,
    "parseWidgets": false
};
</script>
<script type="text/javascript" src="/theme/META-INF/dojo/dojo.js"></script>
<script type="text/javascript" src="/theme/META-INF/json/json.js"></script>
<script type="text/javascript" src="/theme/META-INF/prototype/prototype.js"></script>
<script type="text/javascript" src="/theme/META-INF/com_sun_faces_ajax.js"></script>
<script type="text/javascript">
dojo.hostenv.setModulePrefix("webui.suntheme", "/theme/com/sun/webui/jsf/suntheme/javascript");
dojo.require('webui.suntheme.*');
</script>
<link id="sun_link5" rel="stylesheet" type="text/css" href="/resource/css/css_ns6up.css" />

</head>

<body id="body3" class="LogBdy" focus="loginform.j_username" style="background-color: #4A5C6A;">
    <div id="header"class="LogTopBnd" style="height: 30px;"></div>
    <div class="middle">
        <div class="plugincontent" style="width1: 1px; visibility: hidden;">

<div style="height: 435px;background: #ffffff url(/resource/community-theme/images/login-backimage-open.png);
    background-repeat:no-repeat;background-position:left top; width: 720px; margin: auto; margin: auto; -webkit-border-radius: 10px; -moz-border-radius: 10px; border-radius: 10px;">
    <div style="width: 460px; padding-top: 160px; margin-left: 310px;">
<img id="sun_image11" src="/resource/community-theme/images/login-product_name_open.png;jsessionid=4a7be4a466b5690d43f3878b885b" alt="Payara Server Open Source Edition" height="42" width="329" border="0" />
        <form method="POST" class="form" name="loginform" action="j_security_check">
        <table role="presentation">
        <tr>
            <td><label for="Login.username" style="font-weight: bold; color: #002C3E;">User Name:</label></td>
            <td><input type="text" name="j_username" id="Login.username" tabindex="1" value="admin"></td>
        </tr>
        <tr>
            <td><label for="Login.password" style="font-weight: bold; color: #002C3E;">Password:</label>
            <td><input type="password" name="j_password" id="Login.password" tabindex="2">
        <tr>
            <td colspan="2" align="center">
                <input type="submit" class="Btn1"
                    value="Login"
                    title="Log In to Payara Administration Console" tabindex="3"
                    onmouseover="javascript: if (this.disabled==0) this.className='Btn1Hov'"
                    onmouseout="javascript: if (this.disabled==0) this.className='Btn1'"
                    onblur="javascript: if (this.disabled==0) this.className='Btn1'"
                    onfocus="javascript: if (this.disabled==0) this.className='Btn1Hov'"
                    name="loginButton" id="loginButton">
                    <input type="hidden" name="loginButton.DisabledHiddenField" value="true" />
            </td>
            </tr>
            </table>
        </form>
    </div>
</div>

            <script type="text/javascript">
                if (true) {
                    //submitAndDisable(document.getElementById('loginButton'), 'Login');
                    document.getElementById('loginButton').form.submit();
                    //document.getElementById('loginButton').form.autocomplete="off";
                }
            </script>
        </div>
    </div>
    <div class="footer"
        style="color: #fff;">
        <div id="copyright" style="width: 720px; margin-left: auto; margin-right: auto; padding: 5px; text-align: center">
            <span>Copyright © 2005, 2014, Oracle and/or its affiliates. Payara Portions Copyright © 2020 Payara Foundation. All rights reserved. Payara and Payara Server are trademarks of Payara Foundation.  Oracle and Java are registered trademarks of Oracle and/or its  affiliates. Other names may be trademarks of their respective owners.</span>
        </div>
    </div>
    <script src="/resource/js/cj.js"></script>
</body>
</html>

**Comment by JonasCir** [comment]
Hi @LCHARRE! Try 172.18.0.5:**6080**. Head over to the REST [documentation](https://github.com/hzi-braunschweig/SORMAS-Project/tree/development/sormas-rest) to see how to interact with the API. If you have further questions, please comment!

**Comment by MateStrysewske** [comment]
@JonasCir is right here. 6048 is the **admin** port; if you want to communicate with the application (via the UI or the ReST interface), you have to target port 6080.
